src/components/Details/DetailObject.tsx

```diff
@@ -35,7 +38,15 @@ class DetailObject extends React.Component<DetailObjectProps> {
     return this.props.labels.indexOf(name) > -1;
   }
 
-  buildList(name: string, value: any, isLabel: boolean): any {
+  canDisplay(name: string) {
+    return this.props.exclude == null || !this.props.exclude.includes(name);
+  }
+
+  buildList(name: string, value: any, isLabel: boolean, depth: number): any {
```

Comment:
Perhaps a comment here or at the top of the class just indicating that validations are rendered at first level can help to understand the new "depth" parameter.